src/index.js

```diff
@@ -32,22 +32,11 @@ async function getTemplatesPath(templateName = null) {
 }
 
 async function getTemplateOption() {
-  const templateArg = args.t || args.template
+  const templateArg = config.t || config.template || config.templateName
   if (templateArg) {
     return getTemplatesPath(templateArg)
   }
 
-  const { template } = await inquirer.prompt([
```

Comment:
These lines of code need in case the template is not specified in the config.
The behavior is expected because one of the features of the tool is to use templates from the community.
<img width="278" alt="screen shot 2018-11-20 at 23 39 20" src="https://user-images.githubusercontent.com/7335613/48807399-b1587900-ed1d-11e8-8c17-c9fa5eb6ef9c.png">

Could you please commit again with these lines of code?
Thank you so much man for your contribution 👍

Reply:
Yeah, no problem!